lib/calc_graph.py

```diff
@@ -148,33 +173,34 @@ def find_next_unset_child(self):
                 return None
 
     def to_dict(self):
-        data = {'op': self.op.op_name, 'value': self.value.to_dict()}
+        data = {'op': self.op.op_name if self.op else CGOperator.op_name, 'value': self.value.to_dict()}
         if self.children:
             data['children'] = [c.to_dict() for c in self.children]
         return data
 
+    def op_calc(self, children, input_val):
+        return (self.op.calc(children, input_val)
+                if self.op else CGOperator.static_calc(
```

Comment:
imho, default op=CGOperator will be more clear than considering CGNodes with possible op=None